src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitWorkerService.java

```diff
@@ -128,7 +130,8 @@ protected void saveResult(Network network, AbstractResultContext<VoltageInitRunC
     }
 
     @Override
-    protected void postRun(VoltageInitRunContext runContext, OpenReacResult result) {
+    protected void postRun(VoltageInitRunContext runContext, AtomicReference<ReportNode> rootReportNode, OpenReacResult result) {
+        super.postRun(runContext, rootReportNode, result);
```

Comment:
the below code enrich the report while postRun send the report to report server, in my opinion, super.postRun should be called at the end of method